docassemble/Exp102/data/questions/EXP102.yml

```diff
@@ -339,8 +289,9 @@ fields:
       - I understand and wish to continue
     validation messages:
       minlength: |
-        You cannot continue unless you confirm that you have read the information above.        
+        You cannot continue unless you confirm that you have read the information above.  
 section: i_understand
+#continue button field: Instructions
```

Comment:
```suggestion
continue button field: Instructions
```

Reply:
This was all I could find from the new changes in a very quick run-through. The old code gave me trouble for not entering both phone numbers, but the new changes don't have anything to do with that.